spec/fixtures/custom_nginx.template

```diff
@@ -140,7 +140,7 @@ http {
         listen $(entry.listener);
 > end
 
-        error_page 400 404 408 411 412 413 414 417 494 /kong_error_handler;
+        error_page 400 404 408 411 413 414 417 494 /kong_error_handler;
```

Comment:
For un-mitigated version: if we do not remove 412 from test template, it will cause segfault(signal 11): https://github.com/Kong/kong/actions/runs/9625903064/job/26551305748